src/GovUk.Education.ExploreEducationStatistics.Data.Services/ResultSubjectMetaService.cs

```diff
@@ -101,14 +101,18 @@ public class ResultSubjectMetaService : IResultSubjectMetaService
                     _logger.LogTrace("Got Filters in {Time} ms", stopwatch.Elapsed.TotalMilliseconds);
                     stopwatch.Restart();
 
-                    var footnoteViewModels = await GetFilteredFootnoteViewModels(releaseId, filterItems.Select(fi => fi.Id).ToList(), query);
-                    _logger.LogTrace("Got Footnotes in {Time} ms", stopwatch.Elapsed.TotalMilliseconds);
-                    stopwatch.Restart();
-
                     var indicatorViewModels = GetIndicatorViewModels(query, releaseSubject);
                     _logger.LogTrace("Got Indicators in {Time} ms", stopwatch.Elapsed.TotalMilliseconds);
                     stopwatch.Restart();
 
+                    var footnoteViewModels = await GetFootnoteViewModels(
+                        releaseId: releaseId,
+                        subjectId: query.SubjectId,
+                        filterItemIds: filterItems.Select(fi => fi.Id).ToList(),
+                        indicatorIds: indicatorViewModels.Select(i => i.Value).ToList());
```

Comment:
Cool - good to have those extra params handy.  How come we've not had to pass indicator ids before?

Reply:
It's because we previously passed `query` and then got the id's from `queryContext.Indicators`, but see the comment on the PR from @ntsim on this one starting "There's potentially a bit of a bug around the indicatorIds that get passed into this method"... for an explanation on the bug this had.

Basically the table tool forces a selection of indicators currently, but if it didn't and the list is empty the table result contains all indicators, as does the meta data.

With an empty `queryContext.Indicators` we would be getting footnotes that only apply to no indicators and miss all footnotes that apply to specific indicators if there are any.

I think we should be able to see this bug in prod currently if we query it via the API, although I've not double checked.

Reply:
Cool!